docs/docs/documentation/basics.md

```diff
@@ -0,0 +1,303 @@
+
+In the example, we only provide a ``training_ds`` which is of course not recommended for most use cases, where you need at least a validation set and desirably a test set. We also do not set anything related to the training process, such as optimizer, learning rate, epochs and so on. The library provides basic defaults for this just to get started. When further experimenting, you will probably need to use a trainer, configured with a [TrainerConfiguration](../api/biome/text/configuration.md#trainerconfiguration) object.
+
+## Using pre-trained pipelines
+
+As mentioned above, ``model.tar.gz`` files resulting from a training run contain everything you need to start using your trained pipeline.
+
+Let's see how:
+
+```python
+pipeline = Pipeline.from_pretrained('path_to_store_training_run_output/model.tar.gz')
+```
+After loading the pipeline, you can use it for predictions:
+
+```python
+pipeline.predict(text='Good movie indeed!')
+```
+
+### Predict
+
+The [predict](../api/biome/text/pipeline.md#predict) method for our pipeline will be aligned to match our model input features (`text` in this case). Models such as the `RecordClassifier` can be used to define more fine-grained input features, such as for example an email classifier with two fields `subject` and `body`:
+```python
+pipeline.predict(subject='Hi!', body='Hi, hope you are well..')
+```
+
+### Explain
+Another thing we can do with a trained pipeline is to use the [explain](../api/biome/text/pipeline.md#explain) method to get the attribution of each token by means of [integrated gradients](https://arxiv.org/abs/1703.01365) for those heads that support it:
+```python
+pipeline.explain(text='Good movie indeed!')
+```
+
+### Serve
+Pipeline use the awesome [FastAPI](https://fastapi.tiangolo.com/) library to give you simple REST endpoint with the [serve](../api/biome/text/pipeline.md#serve) method, which provides methods aligned with your input features (e.g., a method accepting `subject` and `body` parameters):
+
+```python
+pipeline.serve(port=9090)
+```
+
+### Explore
+In order to support users with fine-grained error analysis and empower them to improve their models with informed decisions, the [explore](../api/biome/text/pipeline.md#explore) method launches a UI (inside your notebook if you are using Jupyter).
+
+```python
+pipeline.explore(
+    train_ds,
+    explain=True
+)
+```
+
+This search-based UI can help:
+
+ - to quickly identify difficult examples in the training, validation, test or any other data set.
+ - to inform users about corner cases and which labels need more examples.
+ - to find the values for confidence thresholds to use the model in production.
+ - to help domain experts understand the strengths and limitations of the model.
+ - in summary, get a sense of the model interacting with data.
+
+:::tip
```

Comment:
```suggestion
:::tip Tip
```

This was a comment by Leire to override the default "TIP" header (all capital) to "Tip"